enterprise/cmd/worker/internal/batches/workers/batch_spec_workspace_creator.go

```diff
@@ -32,17 +32,11 @@ type batchSpecWorkspaceCreator struct {
 
 // HandlerFunc returns a workeruitl.HandlerFunc that can be passed to a
```

Comment:
While we're here :shrug:

```suggestion
// HandlerFunc returns a workerutil.HandlerFunc that can be passed to a
```